package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "PockyBot",
-	"version": "1.5.1",
+	"version": "1.6.2",
```

Comment:
Technically if we are following semver, we should bump once per release, but I'm fine with this anyway.

Reply:
Yeah good point. But uhh, are we able to deploy during the development process without bumping the patch version?

Reply:
yes that is possible @jimcullenaus